src/js/main.js

```diff
@@ -96,6 +96,8 @@ function lineSource(centers, distanceInKm) {
   );
  });
 
+ console.log(features)
```

Comment:
whoops... kindly remove this @nerdlet I forgot 🙈

Reply:
Hehehe...its fine, I'll remove it